charts/k8ssandra/values.yaml

```diff
@@ -114,9 +114,9 @@ repair:
 backupRestore:
   medusa:
     # If enabled, bucketName and bucketSecret must be defined.
-    enabled: false
+    enabled: true
     # image The Medusa image which is built from https://github.com/thelastpickle/cassandra-medusa/tree/master/k8s.
-    image: k8ssandra/medusa:c5fefc4a3b4c
+    image: localhost:5000/burmanm/medusa:latest
```

Comment:
I know that this is just for dev/testing, but it would be better to keep this as the actual default and override it. It'll make reviewing a little easier.